Makefile

```diff
@@ -50,3 +66,4 @@ clean:
 	$(MAKE) -C topic-controller		clean
 	$(MAKE) -C riff-cli				clean
 
+dev-setup: namespace development-kafka clean build dockerize kubectl-apply
```

Comment:
I'm not too keen on so many fine grained tasks that are only aliases for one kubectl command. What about moving the namespace and kafka parts inside kubectl-apply?

Reply:
I don't mind at all. I figured I'd start with a straight translation.

Would it be worth renaming `kubectl-apply` in that case? (To what I'm not sure, as I haven't had my 2nd coffee yet)

Reply:
`dev-setup`? then it neatly mirrors `teardown`

Reply:
Works for me. I'll push it up in a few.